tensorflow/go/graph.go

```diff
@@ -128,12 +128,7 @@ func (g *Graph) ImportWithOptions(def []byte, options GraphImportOptions) error
 	// with "go vet" till https://github.com/golang/go/issues/17201 is
 	// resolved.
 	buf.length = C.size_t(len(def))
-	buf.data = C.malloc(buf.length)
-	if buf.data == nil {
-		return fmt.Errorf("unable to allocate memory")
-	}
-	defer C.free(buf.data)
```

Comment:
You still need this

Reply:
Sorry It was too rough. And I realized that this PR require version 1.13 or later for Go binding. is this better to separate source code and add build constraints go113 ?

Reply:
Why would you need a build constraint for Go 1.13? CBytes was added in Go 1.7.

Reply:
from comment above, go vet seems to reports usage of C.CBytes.

Reply:
It's been fixed since Go 1.8. Like [I said earlier](https://github.com/tensorflow/tensorflow/pull/33546#discussion_r338319596): "golang/go#17201 (and golang/go#17563) have been fixed since late 2016."

Reply:
I got it. I had thought this issue has fixed recently with your change. Thanks.